examples/example-gcp-csm-observability/README.md

```diff
@@ -0,0 +1,43 @@
+gRPC GCP CSM Observability Example
+================
+
+The GCP CSM Observability example consists of a Hello World client and a Hello World server and shows how to configure CSM Observability
+for gRPC client and gRPC server.
+
+## Configuration
+
+`CsmObservabilityClient` takes the following command-line arguments -
+* user  - The name you wish to be greeted by.
+* target - By default, the client tries to connect to target : `xds:///helloworld:50051` and gRPC would use xDS to resolve this target and connect to the server backend. This can be overridden to change the target.
```

Comment:
This starts with "by default". We should start with what it is, and then say what the default value is.

Reply:
Updated.